paimon-format/src/test/java/org/apache/paimon/format/avro/AvroBulkFormatTestUtils.java

```diff
@@ -35,8 +35,12 @@ public class AvroBulkFormatTestUtils {
             (RowType)
                     RowType.builder()
                             .fields(
-                                    new DataType[] {DataTypes.STRING(), DataTypes.STRING()},
-                                    new String[] {"a", "b"})
+                                    new DataType[] {
+                                        DataTypes.STRING(),
+                                        DataTypes.STRING(),
+                                        DataTypes.TIMESTAMP_WITH_LOCAL_TIME_ZONE(3)
```

Comment:
If take `localTimestampMicros` into consideration, it's better to add a test for > 3.